keps/sig-architecture/3138-increase-reliability-bar/README.md

```diff
@@ -0,0 +1,415 @@
+also figure out a strategy for how to backfill those standards to already
+existing features and tests. However, the details of that are out of scope
+for this document.
+
+
+### [Milestone 3] Addressing long-standing reliability issues
+
+This category covers system issues that cannot be easily fixed with 1 or 2 PRs,
+but rather require significant changes in the system (we can probably
+characterize them as requiring a KEP). A good recent example of something
+in this category may be CronJob (which was staying in Beta for a long time
+due to them).
+
+Given the nature of the problem, as well as the fact that many of them
+exist in Kubernetes for years, we can’t suddenly block contributions from
+a SIG until the problem is addressed. But we really need a way to ensure
+that those will get addressed.
+
+As a result we propose a similar pattern to the one for flaky test:
+1. The initial list of such issues will be prepared by WG Reliability and
+   presented for approval to SIG Architecture (not started yet). Over time,
+   new issues in this category may be added to the list (again with SIG
+   Architecture approval), e.g. as a result of investigations/debugging of
+   test flakes.
+1. For each such issue, we will open a tracking issue in the k/k repo and
+   tag it appropriately with kind/reliability.
+1. From then, a SIG has two releases to present an approved and implementable
+   KEP addressing the problem with a clear execution plan on it (by default
+   Alpha in the upcoming release (N), Beta in the next release (N+1) and
+   Stable in N+3).
+1. If a particular SIG will have more than N (say N=2) such issues opened,
+   the SIG may limit themselves to work on any chosen 2 issues.
+1. If either the KEP itself or its execution will fall over the expected
+   timelines, similarly as above this will block any Alpha or Beta
+   graduations for features owned by this SIG.
+
+
+### [Milestone 4] New reliability investments
+
+This category covers investments that we should make towards increasing the
+quality and/or reliability of Kubernetes. This area includes (but is not
+limited to):
+- creating new types of tests (e.g. chaos tests)
+  - New tests that demonstrate a real issue will be merged and run in their
+   dedicated test suite. Their results will prove the issue from [Milestone 3]
+   category and will be prioritized together with other issues in that category.
+- features to harden Kubernetes (Priority & Fairness is a good current example)
+- architectural improvements leading to better reliability
+
+The list of such investments will be created by the Reliability WG as one of
+the artifacts they should produce.
+
+Given this category is generally quite similar to the previous one, we’re
+going to use the exact same process (including SIG Architecture approval)
+for addressing issues from that list.
+
+
+### Test Plan
+
+N/A
+
+
+### Graduation Criteria
+
+We're going to proceed with this process.
```

Comment:
This feels in conflict of the above idea of doing a dry-run. I think we need to know how we would determine what success looks like for each milestone. Also by saying "milestone" are we implying that they are happening in succession? If so this could be laid out right here in Graduation Criteria. For example if Milestone 1 is completed with x% "success" (to be defined) and participation then 2... etc...